jekyll/_cci2/hello-world-windows.md

```diff
@@ -120,7 +120,7 @@ workflows:
 
 From here we will use the version 2.1 syntax to discuss using the Windows executor, but if you're using Server, you can follow along with the executor definition syntax described above.
 
-## Specifying a Shell with the Windows Executor
+## Specifying a Shell with the Windows executor
```

Comment:
```suggestion
## Specifying a shell with the Windows executor
```
We don't capitalize "executor" in "Windows executor" elsewhere on the page so I don't think it needs to be capitalized here. We also don't need to capitalize "shell" here